src/main/app/controller/GetController.ts

```diff
@@ -17,11 +17,11 @@ export class GetController {
     const commonLanguageContent = commonContent[req.session.lang] || commonContent['en'];
     const commonPageContent = this.content.common || {};
 
-    const errors = req.session.errors || [];
+    const sessionErrors = req.session.errors || [];
```

Comment:
this seems a bit unnecessary now but okay

Reply:
its because in content, errors already exist so i thought ill change it here, if we are going to copy the content over, itll reduce mistakes but we can change it in the content as well.

Reply:
I understand